app/routes.py

```diff
@@ -140,6 +141,19 @@ def update_election(link):
 
         db.session.commit()
 
+        # the scheduler
+        try:
+            # remove the previous scheduled event then recreate with form data
+            scheduler.remove_job(id="start " + election.name)
```

Comment:
Hey, can we use the Election.id instead of Election.name since name can change, but ID cannot?